src/main/content/_assets/js/collections.js

```diff
@@ -21,10 +21,25 @@ function getCollectionData(instanceName){
         return;
     }
 
+    let collectionJSON = null;
+    let cliVersion = null;
+
     return fetch(`/api/auth/kabanero/${instanceName}/collections/list`)
         .then(function(response) {
             return response.json();
         })
+        .then(function(data) {
+            collectionJSON = data;
+            return fetch(`/api/auth/kabanero/${instanceName}/collections/version`)
```

Comment:
since there's really no relation between `/${instanceName}/collections/list` and `/${instanceName}/collections/version` we should separate this out into its own function for organization purposes.

Reply:
then call the new method in the promise chain after `handleInitialCLIAuth` in the document ready function